src/main/resources/workerScripts/fromMultiplePriorityQueues.lua

```diff
@@ -15,7 +16,7 @@ local OPTIONAL_SPACE_SEPARATOR = '%s*'
 local NEXT_QUEUE_REGEX = QUEUE_NAME_CAPTURING_REGEX .. OPTIONAL_COMMA_SEPARATOR .. OPTIONAL_SPACE_SEPARATOR
 
 for q in queues.gmatch(queues, NEXT_QUEUE_REGEX) do
-    local queueName = 'resque:queue:' .. q
```

Comment:
Default namespace is `resque`?
Just make sure this change is backwards compatible

Reply:
@ofirnk Yes, tested by enqueuing using the old client and reading using the new client. And yes, `resque` was the default namespace being used.